sources/packages/backend/apps/api/src/route-controllers/application/_tests_/application.students.controller.getInProgressApplicationDetails.e2e-spec.ts

```diff
@@ -0,0 +1,277 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import {
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getStudentToken,
+  getStudentByFakeStudentUserType,
+  FakeStudentUsersTypes,
+} from "../../../testHelpers";
+import {
+  createFakeCRAIncomeVerification,
+  createFakeSupportingUser,
+  createE2EDataSources,
+  E2EDataSources,
+  saveFakeApplication,
+  createFakeStudentAssessment,
+} from "@sims/test-utils";
+import {
+  Student,
+  StudentAssessmentStatus,
+  SupportingUserType,
+} from "@sims/sims-db";
+import { SuccessWaitingStatus } from "../models/application.dto";
+
+describe("ApplicationStudentsController(e2e)-getApplicationProgressDetails", () => {
+  let app: INestApplication;
+  let student: Student;
+  let db: E2EDataSources;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+    student = await getStudentByFakeStudentUserType(
+      FakeStudentUsersTypes.FakeStudentUserType1,
+      dataSource,
+    );
+  });
+
+  it("Should throw not found error when application is not found.", async () => {
+    // Arrange
+
+    const endpoint = `/students/application/99999999/in-progress`;
+    const token = await getStudentToken(
+      FakeStudentUsersTypes.FakeStudentUserType1,
+    );
+
+    // Act/Assert
+    await request(app.getHttpServer())
+      .get(endpoint)
+      .auth(token, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.NOT_FOUND);
+  });
+
+  it("Should get application in-progress details when the student does not have a supporting user.", async () => {
```

Comment:
These E2E tests are not part of the ticket. 

I would advise to keep the efforts of E2E relevant to the user story.